packages/gatsby-plugin-facebook-analytics/src/gatsby-ssr.js

```diff
@@ -4,37 +4,41 @@ import { stripIndent } from "common-tags"
 export const onRenderBody = ({ setPostBodyComponents }, pluginOptions) => {
   const {
     appId,
-    includeInDevelopment = false,
+    cookie = false,
     debug = false,
+    includeInDevelopment = false,
     language = `en_US`,
+    version = `v2.12`,
```

Comment:
Are there any breaking change from 2 - 3? Otherwise, is there a reason we wouldn't just use the latest and greatest?

It's also just fine to make a breaking change in the plugin itself to accommodate using 3.3.

Reply:
It is kinda nice to keep it _for sure_ not a breaking change though--so I appreciate the effort here to enable that!

Reply:
I'm new to Facebook Analytics myself. However, looking at the Facebook JavaScript SDK reference documents and the plugin code, I think you are correct and that it should be okay to bump the default to the latest version 👍

Reply:
I've implemented the change to set the default version at v3.3 👍

Reply:
Do you have a Facebook Analytics account you can test the changes with? Doesn't seem like we'd have broken anything here--but worth validating!

Otherwise one of us can validate manually early next week!

Reply:
I think we're probably good -> https://developers.facebook.com/docs/graph-api/changelog/breaking-changes/

Reply:
@DSchau Yes, I followed the [How to Make a Reproducible Test Case](https://www.gatsbyjs.org/contributing/how-to-make-a-reproducible-test-case/) to setup a test site and then used the gatsby-dev-cli to pull in the changes I had made and set the configuration to point to a test Facebook App and have seen some data come through.
<img width="1129" alt="Screen Shot 2019-05-24 at 23 04 08" src="https://user-images.githubusercontent.com/3592472/58358786-5ba18c80-7e78-11e9-9d0f-a6f84ff6f508.png">

Reply:
Nicely done!! Let's get this merged, then. Thanks so much!

Reply:
It was a pleasure to contribute and thank you for the feedback and reviewing so quickly 😄